### PR TITLE
Client SDK: Added verifyKeypair()

### PR DESCRIPTION
**Pull request by MartinMinkov**
- Exposes `validKeypair()` in client_sdk.ml to the SDK as `verifyKeypair()`
- Added a new test for `verifyKeypair()`

### CODE REVIEW

File: frontend/client_sdk/src/CodaSDK.d.ts [hunk 1]
```diff
@@ -45,6 +45,15 @@ export declare const genKeys: () => keypair;
   * @returns A public key
   */
 export declare const derivePublicKey: (privateKey: privateKey) => publicKey;
+/**
```
Comment (psteckler) [inline frontend/client_sdk/src/CodaSDK.d.ts:48]: should this mention that an exception is raised if the keypair is invalid?

File: frontend/client_sdk/src/CodaSDK.re [hunk 2]
```diff
@@ -73,6 +73,20 @@ external publicKeyOfPrivateKey: (codaSDK, privateKey) => publicKey =
 let derivePublicKey = (privateKey: privateKey) =>
   publicKeyOfPrivateKey(codaSDK, privateKey);
 
+[@bs.send] external validKeypair: (codaSDK, keypair) => bool = "validKeypair";
+/**
```
Comment (psteckler) [inline frontend/client_sdk/src/CodaSDK.re:77]: mention the exception?

**Review by psteckler — CHANGES_REQUESTED** [review]
should mention that `verifyKeypair` raises on invalid keypair

File: frontend/client_sdk/src/CodaSDK.d.ts (hunk 1 above)
Comment (MartinMinkov) [reply frontend/client_sdk/src/CodaSDK.d.ts:48]: good catch, updated!

**Review by psteckler — APPROVED** [review]

**Comment by aneesharaines** [comment]
!approved-for-mainnet

**Review by lk86 — APPROVED** [review]

**Pull request merged**